### PR TITLE
VSS-701-AJ Added header to contact page

### PR DESCRIPTION
**Pull request by xDAnkit**
- Added header to contact
- Added header CSS to contact
- Added `.gitignore` file to ignore `.vscode` setting

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -5,6 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>About</title>
+    <style>
+      body {
+        margin: 0;
```
Comment (std447) [inline about.html:10]: @xDAnkit do we need to add margin here.
Can we do it in common styling

**Review by std447 — CHANGES_REQUESTED** [review]
Please check my feedbacks.

File: about.html (hunk 1 above)
Comment (xDAnkit) [reply about.html:10]: Hey @std447 , as we're not aligned to create a separate file hence we decided to go with internal CSS>

Comment (std447) [reply about.html:10]: Done

**Pull request merged**